apps/src/lab2/projects/ProjectManager.ts

```diff
@@ -364,7 +364,13 @@ export default class ProjectManager {
           forceNewVersion
         );
       } catch (error) {
-        this.onSaveFail('Error saving sources', error as Error);
+        let errorToReport: Error;
+        if (error instanceof Error) {
```

Comment:
Out of curiosity in what instances does this actually happen (the thrown error is not instance of an `Error`)?